gh2/csv.py

```diff
@@ -190,19 +196,24 @@ def write_rows(filename, headers, fields, issues, date_format, include_prs,
             issue_data = issue_to_dict(fields, issue, additional_labels)
             if not skip_normalization:
                 issue_data = normalize_sequential_dates(issue_data)
+            if include_milestones:
+                milestone = issue.milestone.title if issue.milestone \
+                    is not None else u'No Milestone'
+                issue_data.update({u'Milestone': milestone})
             writer.writerow(format_dates(issue_data.values(), date_format))
 
 
-def set_headers(repo, labels=None):
+def set_headers(repo, labels=None, milestones=None):
     headers = [
         'ID', 'Link', 'Name', 'Backlog', 'Triage', 'Investigate', 'Approved',
         'Doing', 'Needs Review (Ready)', 'Needs Review (Doing)',
         'Backport (Ready)', 'Backport (Doing)', 'Documentation (Ready)',
         'Documentation (Doing)', 'Pending SHA Update', 'Dev Done',
     ]
     if labels:
-       headers.extend('Label: ' + label.name for label in labels)
-
+        headers.extend('Label: ' + label.name for label in labels)
+    if milestones:
```

Comment:
I'd be inclined to add the milestone before the additional labels. The labels can potentially be a long list and I suspect it would be easier to toggle the milestone if it isn't separated from all the other data by the labels.